android/src/main/java/com/RNTextInputMask/RNTextInputMaskModule.java

```diff
@@ -40,9 +40,9 @@ public void mask(final String maskString,
       final Mask.Result result = mask.apply(
           new CaretString(
               input,
-              input.length()
-          ),
-          true
+              input.length(),
+              new CaretString.CaretGravity.FORWARD(false)
```

Comment:
Api has changed, even though i've switch autocomplete off here it still autocompletes forward. If we switch this to true, then autocomplete causes problems when trying to clear out entered value